prime-router/settings/prod/0037-wa-uat.yml

```diff
@@ -0,0 +1,81 @@
+# Connect WA-PHD to their UAT (aka test) environment.
+# Note: the JKS secret must be set for this connection to complete
+# ./prime multiple-settings set --input settings/prod/0037-wa-uat.yml --env prod
+---
+- name: wa-phd
+  description: Washington Department of Health
+  jurisdiction: STATE
+  stateCode: WA
+  receivers:
+  - name: "elr"
+    organizationName: "wa-phd"
+    topic: "covid-19"
+    translation:
+      type: HL7
+      useBatchHeaders: true
+      nameFormat: washington
+      reportingFacilityName: 7uycso49
+      reportingFacilityId: 1.3.6.1.4.1.38630.2.1.1.519
+      reportingFacilityIdType: ISO
+      receivingApplicationName: WADOHPHRED
+      receivingApplicationOID: 2.16.840.1.113883.3.237.4.2
+      receivingFacilityName: dn1fro00
+      receivingFacilityOID: 1.3.6.1.4.1.38630.2.1.1.19
+      suppressQstForAoe: true
+      phoneNumberFormatting: ONLY_DIGITS_IN_COMPONENT_ONE
+      useTestProcessingMode: true
+    jurisdictionalFilter:
+      - matches(abnormal_flag, A)
+      - "orEquals(ordering_facility_state, WA, patient_state, WA)"
+    qualityFilter: []
+    reverseTheQualityFilter: false
+    deidentify: false
+    timing:
+      operation: "MERGE"
+      numberPerDay: 12
+      initialTime: "01:15"
+      timeZone: "EASTERN"
+      maxReportCount: 500
+    description: ""
+    transport:
+      type: AS2
+      receiverUrl: "https://uat-onehealthport-as2.axwaycloud.com/exchange/ZZOHPUAT"
```

Comment:
Just sanity checking:  this URL is the same for both regular elr and elr-neg.   Is that intended?